languages/tree-sitter-stack-graphs-javascript/src/stack-graphs.tsg

```diff
@@ -3698,6 +3698,15 @@ inherit .containing_class_value
 
 }
 
+(jsx_namespace_name (_) @lhs (_) @rhs)@name {
+  node @name.before_scope
+  node @name.after_scope
+
+  edge @lhs.before_scope -> @name.before_scope
+  edge @rhs.before_scope -> @lhs.after_scope
```

Comment:
I'm not 100% sure about this one. My thinking is that the namespace (`@lhs`) might resolve to a node in which `@rhs` can then be resolved 🤷🏻

Reply:
…about this specific line, I should say.